### DISCUSSION TITLE
[Question] Bytecode version?

### DISCUSSION THREAD

**Discussion by MCJEModder2026** [cc-tweaked/CC-Tweaked#1474] [Q&A] [answered]
What lua version is the lua bytecode from? Also is the bytecode different from what can be made with regular lua from the website?

**Comment by piprett** [comment]
This might help you?:
https://tweaked.cc/reference/feature_compat.html

**Comment by MCJEModder2026** [comment]
No i mean if i compiled code with luac could i run it in cc tweaked

**Comment by SquidDev** [comment, marked as answer]
CC:T currently uses Lua 5.1's bytecode format. However, I'd strongly recommend not relying on this - it will change in the future when we change Lua versions.

**Reply by MCJEModder2026** [reply]
It seems the only difference currently is size_t, on my machine it is 8 but in cc it is 4
I used chunkspy to cross compile and it worked
